core/src/test/scala/org/allenai/common/EnumSpec.scala

```diff
@@ -4,15 +4,11 @@ import org.allenai.common.testkit.UnitSpec
 
 import spray.json._
 
-class EnumSpec extends UnitSpec {
+import java.io.{ FileOutputStream, FileInputStream }
```

Comment:
https://github.com/daniel-trinh/scalariform/issues/66

On Wed, Nov 18, 2015 at 8:34 AM, Mark Schaake <notifications@github.com>
wrote:

> In core/src/test/scala/org/allenai/common/EnumSpec.scala
> <https://github.com/allenai/common/pull/159#discussion_r45222945>:
>
> > @@ -4,15 +4,11 @@ import org.allenai.common.testkit.UnitSpec
> >
> >  import spray.json._
> >
> > -class EnumSpec extends UnitSpec {
> > +import java.io.{ FileOutputStream, FileInputStream }
>
> Sticking to our import formatting standards I think is a good policy for
> code readability (especially code review and approaching code that you
> haven't seen before). There is a real cost without simple automated tooling
> (agreeing with Cristian - evidence is the cost of having this conversation
> which is one that has been had many times at AI2). I think we should either
> finally invest in automating import formatting or decide to drop the import
> formatting standard. My vote would be for investing in automation. We
> already have Jesse's git commit hook scalariform formatter. Would be great
> if we could hook into that (no pun intended) with an auto import formatter.
> Not sure how hard it would be to write a tool that did this. I've written
> some elisp functions
> <https://github.com/markschaake/emacs-conf/blob/master/ai2-imports.el>
> that do this for me Emacs. Could be ported to scala easily and implemented
> as an SBT plugin (for example).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/allenai/common/pull/159/files#r45222945>.
>